test/FunctionalTests/Microsoft.OData.Client.Tests/ALinq/DataServiceContextMethodMockSampleTest.cs

```diff
@@ -0,0 +1,56 @@
+﻿//---------------------------------------------------------------------

```

Comment:
Maybe this checkin is meant for a different PR? It doesn't seem related to #1332